src/index.html

```diff
@@ -5,9 +5,46 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+
+      <img
+        src="images/imac.jpeg"
+        alt="Apple A1419 iMac 27 Retina 5K Monoblock(MNED2UA/A)"
+        class="card__img"
+      >
+
+      <h2 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <p class="card__product-code">Product code: 195434</p>
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__rating-reviews">Reviews: 5</p>
+      </div>
+      <div class="card__prices">
+        <p class="card__prices-text">Price:</p>
+        <p class="card__prices-number">$2,199</p>
+      </div>
+      <a
+        href="#"
+        class="card__btn"
+        data-qa="hover">
+        Buy
```

Comment:
```suggestion
      <a
        href="#"
        class="card__btn"
        data-qa="hover"
      >
        Buy
```
closing bracket should be on the same level as opening one, 14'th point in checklist